### PR TITLE
Update Style-and-Tone.md

### PR DESCRIPTION
**Pull request by pljones**
**Short description of changes**

Add "Server List" to "Standard terms".

**Context: Fixes an issue? Related issues**

No but see https://github.com/jamulussoftware/jamulus/pull/3017#discussion_r1148594065

**Status of this Pull Request**

To be reviewed.

**What is missing until this pull request can be merged?**

To be reviewed.

**Does this need translation?**

Yes.

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

File: contribute/en/Style-and-Tone.md [hunk 1]
```diff
@@ -89,16 +89,18 @@ Jamulus is “Free and Open Source (FOSS)” (not “free software” or “open
...
 
 “Server” When capitalized, refers to an instance of Jamulus running in server mode. When lowercase as _server_, this refers to the computer that runs the Server (e.g. "A Server running on an AWS server"). Not to be confused with Directory.
 
 “Directory” The term for a type of Server that a Client uses to get a list of Servers from. Avoid the use of the term _Directory Server_ because it may be confusing in the presence of _Server_ on its own.
 
-“Country/Region” Keep in mind that some areas of the world have a controversial (political) status. If possible, be generic and remain neutral. Instead of just saying country, use _Country/Region_ or _Location_.
-
 Registration: When a Server is configured in Registered mode, it will be _listed_ when successfully registered by a Directory. Note that if a Directory is full, a Registered Server will not be _listed_ because it has not been successfully _registered_. Note in this case we prefer to say, "Register with a Directory".
 
+“Server List” This is the list of Servers maintained by a Directory.  A Server registers with a Directory to be _listed_ in that Directory’s server list.
```
Comment (pljones) [inline contribute/en/Style-and-Tone.md]:
```suggestion
“Server List” This is the list of Servers maintained by a Directory.  A Server registers with a Directory to be _listed_ in that Directory’s _server list_.
```

**Comment by ignotus666** [comment]
This doesn't get translated - they're guidelines for writing the original docs in English.

**Comment by ann0see** [comment]
Agree. Could you please rebase it to release and point the PR to release? We can then publish it directly.

**Review by ann0see — APPROVED** [review]

**Review by ann0see — CHANGES_REQUESTED** [review]
As above.

**Comment by pljones** [comment]
Replaced by #911

**Pull request closed without merging**